### PR TITLE
Add scala 2.13.0-M5 support

### PR DESCRIPTION
**Pull request by alexarchambault**

### CODE REVIEW

**Comment by SethTisue** [comment]
```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: com.lihaoyi#utest_2.13.0-M5;0.6.5: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
```

https://github.com/lihaoyi/utest/issues/183

**Comment by SethTisue** [comment]
I think some source changes are needed. if I try this branch in the 2.13 community build, I get

```
[acyclic] [error] /Users/tisue/community.213/target-0.9.14/project-builds/acyclic-0f3fc9aaae33492b160c91151afbc48c5ebc4da5/src/main/scala/acyclic/plugin/PluginPhase.scala:96: type mismatch;
[acyclic] [error]  found   : scala.collection.MapView[acyclic.plugin.Value,Seq[PluginPhase.this.global.Tree]]
[acyclic] [error]  required: Map[acyclic.plugin.Value,Seq[PluginPhase.this.global.Tree]]
[acyclic] [error]                      .mapValues(_.map(_._2))
[acyclic] [error]                                ^
[acyclic] [error] /Users/tisue/community.213/target-0.9.14/project-builds/acyclic-0f3fc9aaae33492b160c91151afbc48c5ebc4da5/src/main/scala/acyclic/plugin/PluginPhase.scala:114: type mismatch;
[acyclic] [error]  found   : scala.collection.MapView[acyclic.plugin.Value,Seq[PluginPhase.this.global.Tree]]
[acyclic] [error]  required: Map[acyclic.plugin.Value,Seq[PluginPhase.this.global.Tree]]
[acyclic] [error]                .mapValues(_.flatMap(_._2))
[acyclic] [error]                          ^
[acyclic] [error] /Users/tisue/community.213/target-0.9.14/project-builds/acyclic-0f3fc9aaae33492b160c91151afbc48c5ebc4da5/src/main/scala/acyclic/plugin/PluginPhase.scala:145: trait SortedSet takes type parameters
[acyclic] [error]           cycleInfo.map{ case (a, b) => a -> b.map(_.pos.line).to[SortedSet]}
[acyclic] [error]                                                                   ^
[acyclic] [error] /Users/tisue/community.213/target-0.9.14/project-builds/acyclic-0f3fc9aaae33492b160c91151afbc48c5ebc4da5/src/main/scala/acyclic/plugin/PluginPhase.scala:145: missing argument list for method to in trait IterableOnceOps
[acyclic] [error] Unapplied methods are only converted to functions when a function type is expected.
[acyclic] [error] You can make this conversion explicit by writing `to _` or `to(_)` instead of `to`.
[acyclic] [error]           cycleInfo.map{ case (a, b) => a -> b.map(_.pos.line).to[SortedSet]}
[acyclic] [error]                                                                  ^
[acyclic] [info] No documentation generated with unsuccessful compiler run
[acyclic] [error] four errors found
[acyclic] [error] /Users/tisue/community.213/target-0.9.14/project-builds/acyclic-0f3fc9aaae33492b160c91151afbc48c5ebc4da5/src/main/scala/acyclic/plugin/PluginPhase.scala:96: type mismatch;
[acyclic] [error]  found   : scala.collection.MapView[acyclic.plugin.Value,Seq[PluginPhase.this.global.Tree]]
[acyclic] [error]  required: Map[acyclic.plugin.Value,Seq[PluginPhase.this.global.Tree]]
[acyclic] [error]                      .mapValues(_.map(_._2))
[acyclic] [error]                                ^
[acyclic] [error] /Users/tisue/community.213/target-0.9.14/project-builds/acyclic-0f3fc9aaae33492b160c91151afbc48c5ebc4da5/src/main/scala/acyclic/plugin/PluginPhase.scala:114: type mismatch;
[acyclic] [error]  found   : scala.collection.MapView[acyclic.plugin.Value,Seq[PluginPhase.this.global.Tree]]
[acyclic] [error]  required: Map[acyclic.plugin.Value,Seq[PluginPhase.this.global.Tree]]
[acyclic] [error]                .mapValues(_.flatMap(_._2))
[acyclic] [error]                          ^
[acyclic] [error] /Users/tisue/community.213/target-0.9.14/project-builds/acyclic-0f3fc9aaae33492b160c91151afbc48c5ebc4da5/src/main/scala/acyclic/plugin/PluginPhase.scala:145: trait SortedSet takes type parameters
[acyclic] [error]           cycleInfo.map{ case (a, b) => a -> b.map(_.pos.line).to[SortedSet]}
[acyclic] [error]                                                                   ^
[acyclic] [error] /Users/tisue/community.213/target-0.9.14/project-builds/acyclic-0f3fc9aaae33492b160c91151afbc48c5ebc4da5/src/main/scala/acyclic/plugin/PluginPhase.scala:145: missing argument list for method to in trait IterableOnceOps
[acyclic] [error] Unapplied methods are only converted to functions when a function type is expected.
[acyclic] [error] You can make this conversion explicit by writing `to _` or `to(_)` instead of `to`.
[acyclic] [error]           cycleInfo.map{ case (a, b) => a -> b.map(_.pos.line).to[SortedSet]}
[acyclic] [error]                                                                  ^
[acyclic] [error] four errors found
```

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=0.13.9
+sbt.version=0.13.15
```
Comment (SethTisue) [inline project/build.properties]: better to go all the way to 0.13.17

Comment (alexarchambault) [reply project/build.properties]: Fixed (switched to `1.2.3` at the end…)

**Comment by alexarchambault** [comment]
@SethTisue The compilation errors should be fixed.

**Comment by alexarchambault** [comment]
@lihaoyi Anything we can do to help merge, then release that?

**Comment by lihaoyi** [comment]
@alexarchambault gave you commit access to this, @olafurpg could you help me pass alex the publishing credentials for com.lihaoyi? then you can publish artifacts stuff on your own (he already technically has access to the credentials due to commit access to ammonite/mill, but it's a bit of a pain to extract from travis)

**Comment by alexarchambault** [comment]
Thanks @lihaoyi! I guess I'm going to wait for a +1, then merge (and see with @olafurpg to release).

**Comment by SethTisue** [comment]
scala/make-release-notes@bdf0a0781d6189e9485401e7fb5412680d87a929

**Pull request merged**